cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -35,6 +35,13 @@ export NO_PROXY="localhost,127.0.0.1,169.254.169.254"
 PROXY
 fi
 
+# Configure Amazon Linux 2 instance running in US isolated region.
+. /etc/os-release
+if [[ "${!ID}${!VERSION_ID}" == "amzn2" && "${AWS::Region}" == us-iso* ]]; then
+  configuration_script="/opt/parallelcluster/scripts/patch-iso-instance.sh"
+  [ -f ${!configuration_script} ] && bash ${!configuration_script} "${AWS::Region}"
```

Comment:
With https://github.com/aws/aws-parallelcluster-cookbook/pull/1823 patch you moved the detection of the region into the script itself so the region parameter is useless

Reply:
Fixing in next revision.